api/app/controllers/mno_enterprise/jpi/v1/orga_relations_controller.rb

```diff
@@ -0,0 +1,40 @@
+module MnoEnterprise
+  class Jpi::V1::OrgaRelationsController < Jpi::V1::BaseResourceController
```

Comment:
Authorisation? :policeman:

Reply:
What kind of authorisation ? It's a read only controller.  Would like it still added to ability.rb ?